src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -155,9 +158,8 @@ typedef struct wrapped_rr_closure_arg {
   /* The RR instance related to the closure */
   grpc_lb_policy *rr_policy;
```

Comment:
You're right -- I somehow missed that we were unreffing the RR policy there.  Sorry for the confusion.